src/components/Header/Header.module.css

```diff
@@ -31,8 +31,7 @@
 }
 
 /**
- * Container around sticky header
- * 1) This is added to
```

Comment:
Lol thanks for removing this beginning of a sentence that goes nowhere 😅 I'm really glad we're cleaning this stuff up ❤️